overlord/snapstate/snapstate_test.go

```diff
@@ -599,6 +599,11 @@ func (s *snapmgrTestSuite) TestUpdateMany(c *C) {
 
 	ts := tts[0]
 	verifyUpdateTasks(c, unlinkBefore|cleanupAfter, 3, ts, s.state)
+
+	// check that the tasks are in non-default lane
+	for _, t := range ts.Tasks() {
+		c.Assert(t.Lanes(), DeepEquals, []int{1})
```

Comment:
Should each task be in the same lane? I would expect that each install is in separate one.

Reply:
This test addition is to address previous comment from @pedronis about missing test for already existing case with lanes in updateMany test. And it's ok since this test actually updates a single snap, so we see a single (non-default) lane. I'll however move this change to a separate PR.